### PR TITLE
AC Test Fix - LRAC-9300 | master

### PR DESCRIPTION
**Pull request by andreefarias**
https://issues.liferay.com/browse/LRAC-9300 (test fix ticket)
https://issues.liferay.com/browse/LRAC-9361 (refactor ticket)
https://issues.liferay.com/browse/LRAC-9350 (bug ticket)

### CODE REVIEW

**Comment by andreefarias** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACAssets.path [hunk 1]
```diff
@@ -18,7 +18,7 @@
 </tr>
 <tr>
 	<td>ASSETS_INFO</td>
-	<td>xpath=(//td[contains(@class,'name-cell')][.//*[text()='${key_assetName}']]//following-sibling::td[not(contains(@class,'actions'))])[${index}]</td>
+	<td>//td[contains(@class,'name-cell')][.//*[text()='${key_assetName}']]//following-sibling::td[not(contains(@class,'actions'))][${index}]</td>
 	<td></td>
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACAssets.path:22]: Hi @andreefarias, I don't think this change would be necessary. Why did you change it?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DeleteSegments.testcase [hunk 2]
```diff
@@ -277,6 +283,8 @@ definition {
 				userLastName = "ac",
 				userScreenName = "ac");
 
+			ACUtils.createBlogsAndAddToPage();
+
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DeleteSegments.testcase:287]: Hi @andreefarias, I don't think this change would be necessary

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DeleteSegments.testcase [hunk 3]
```diff
@@ -302,6 +310,8 @@ definition {
 			Navigator.gotoSitePage(
 				pageName = "AC Page",
 				siteName = "Site Name");
+
+			BlogsNavigator.gotoPG(entryTitle = "Blogs AC Title");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DeleteSegments.testcase:314]: Hi @andreefarias, I don't think this change would be necessary

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DeleteSegments.testcase (hunk 2 above)
Comment (andreefarias) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DeleteSegments.testcase:287]: Ok.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DeleteSegments.testcase (hunk 3 above)
Comment (andreefarias) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DeleteSegments.testcase:314]: Ok.

File: portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACAssets.path (hunk 1 above)
Comment (andreefarias) [reply portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACAssets.path:22]: This was from another testcase. I removed it.

**Comment by christianstokes** [comment]
ci:forward
